src/test/java/jimmy/mcgymmy/logic/commands/EditCommandTest.java

```diff
@@ -39,7 +39,8 @@ public void execute_allFieldsSpecifiedUnfilteredList_success() {
                 new CommandParserTestUtil.OptionalParameterStub<>("n", editedFood.getName()),
                 new CommandParserTestUtil.OptionalParameterStub<>("p", editedFood.getProtein()),
```

Comment:
Might want to add some test for editing the date in the future

Reply:
Added